clients/src/main/java/org/apache/kafka/common/security/oauthbearer/OAuthBearerLoginModule.java

```diff
@@ -231,6 +238,8 @@
     private AuthenticateCallbackHandler callbackHandler = null;
     private OAuthBearerToken tokenRequiringCommit = null;
     private OAuthBearerToken myCommittedToken = null;
+    private Map<String, String> extensionsRequiringCommit = null;
```

Comment:
Is there a reason why we shouldn't always use SaslExtensions rather than Map?

Reply:
Actually - no. My bad